ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -1739,6 +1780,111 @@ def reset_device(self, user_inventory: DeviceInventory):
             results.append(result_data)
         return results
 
+    def validate_config(
+        self,
+        user_inventory: DeviceInventory,
+        validate_mode: str = ModuleOptionValues.VALIDATE_MODE_SKIP,
+    ):
+        results = []
+        device_data = {"warnings": [], "errors": []}
+        for device in user_inventory.devices:
+            result_data = CvApiResult(
+                action_name=device.info[self.__search_by] + "_validated"
+            )
+            if device.system_mac is None:
+                continue
+            MODULE_LOGGER.debug(
+                "Device configlets are {0}".format(str(device.configlets))
```

Comment:
Use f-string or with the .format we don't need to specify the str(var).

```
f"Device configlets are {device.configlets}"
```

```suggestion
                "Device configlets are {0}".format(device.configlets)
```

Reply:
Use same technique below as well, where you used str method